docs/source/authoring_flows.rst

```diff
@@ -313,24 +332,22 @@ The method for generating the required tokens is outside the scope of this docum
 Action Execution Monitoring
 ---------------------------
 
-``Action`` states will block waiting until the executed action reaches a
-completion state with status value either ``SUCCEEDED`` or ``FAILED`` or when
-the ``WaitTime`` duration is reached. Within this time interval, the Flow will
-periodically poll the Action to determine if it has reached a completion state.
-The interval between polls increases using an exponential back-off strategy
-(i.e. the amount of time between two polls is a multiple of the interval between
-the previous two polls). Thus, detection of the completion will not be
-instantaneous compared to when the action "actually" completes. And, the longer
-the wait time, the longer the interval between "actual" completion and the poll
-detecting completion may be. This "slop" time is related to both the total run
-time for the Action and the exponential back-off factor increasing the time
-between polls.
-
-When using the Flows service, it is important to remember that this slop time
-can occur. One may observe or receive other notification (such as an email for
-a Globus Transfer) that an Action has completed but the Flows service may not
-poll to discover the same state has been reached. This is an inherent property
-of the system.
+``Action`` states will block until the executed action reaches a
+completion state with status value either ``SUCCEEDED`` or ``FAILED``
+or when the ``WaitTime`` duration is reached. Within this time, the
+Flow will periodically poll the Action to determine if it has reached
+a completion state.  The interval between polls doubles after each
+poll ("exponential back-off") up to a maximum interval between polls
+of 10 minutes. Thus, detection of the completion will not be
+instantaneous compared to when the action "actually" completes and may
+be delayed up to the maximum poll interval of 10 minutes.
+
+It is important to remember that this delay between an Action actually
+being completed and it being recognized by the Flow can occur. A user
+running a flow may observe or receive other notification (such as an
```

Comment:
Should `notification` be plural on this line? Something about this sentence feels wonky, though it may be technically correct.

Reply:
@kurtmckee see if the updated text reads any better...

Reply:
Yep!